client/src/routes/search.tsx

```
@@ -0,0 +1,48 @@
+
+      <InfiniteScroll
+        onLoadMore={!!search.q ? experiencesQuery.fetchNextPage : undefined}
+      >
+        <ExperienceList
+          experiences={
+            experiencesQuery.data?.pages.flatMap((page) => page.experiences) ??
+            []
+          }
+          isLoading={experiencesQuery.isLoading || experiencesQuery.isLoading}
+          noExperiencesMessage={
+            !!search.q ? "No experinces found" : "Search to find experiences"
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)